OptimizelySDK.Tests/OptimizelyTest.cs

```diff
@@ -185,7 +186,23 @@ public void TestValidateInputsInvalidFileJsonValidationSkipped()
         {
             string datafile = "{\"name\":\"optimizely\"}";
             Optimizely optimizely = new Optimizely(datafile, null, null, null, skipJsonValidation: true);
-            Assert.IsTrue(optimizely.IsValid);
+            Assert.IsFalse(optimizely.IsValid);
+        }
+
+        [Test]
+        public void TestErrorHandlingWithInvalidConfigVersion()
```

Comment:
Please split these up into three separate unit tests. Makes it easier to read and determine what actually failed